reconcile-issue/action.yml

```diff
@@ -26,6 +26,10 @@ inputs:
   additionalLabels:
     description: |
       Comma separated list of labels that should be added to Jira issues.
+  autoWatchAccounts:
```

Comment:
nit but `additionalWatchers` might convey the same thing with less confusion.

Reply:
"additional" is a little weird to me, because what is it in addition to?

autoWatchAccounts are the accounts that should be automatically watching the synced Jira.

Reply:
In addition to the bot that is automatically added as watcher. Prefer no confusion :sunglasses: 
If that's true, why not just `watchers`?

Reply:
I'm expecting to add a field that's the opposite polarity. So autoWatchAccounts bring the declarative "these are the accounts that must be watching" and then I plan to add "autoUnwatchAccounts".

In light of that do you have any good field names for both?